src/store/modules/util/index.ts

```diff
@@ -0,0 +1,18 @@
+import actions from './actions'
+import getters from './getters'
+import mutations from './mutations'
+import { Module } from 'vuex'
+import RootState from '@/store/RootState'
+import UtilState from './UtilState'
+
+const utilModule: Module<UtilState, RootState> = {
+    namespaced: true,
+    state: {
+      securityGroups: [],
```

Comment:
Persist security group options, as this information is not something that will change very frequently.

Reply:
Done.